src/run.js

```diff
@@ -177,7 +177,12 @@ const minimalcss = async options => {
     })
 
     await new Promise(async (resolve, reject) => {
-      page.on('pageerror', error => reject(error))
+      let pageerrorCalled = false
```

Comment:
The name isn't great. The *point* of this variable is to **avoid calling `reject()` or `resolve()` more than once**, right? How about something like this:

```javascript
// If anything goes wrong, for example a `pageerror` event or 
// a bad download request (e.g. !response.ok), then remember that 
// we have fulfilled the promise and don't want to call `reject` or `resolve`
// a second time.
let fulfilledPromise = false
```